frontend/src/prompted_readings/analysis_view.js

```diff
@@ -28,14 +28,32 @@ class AnalysisView extends React.Component {
     render() {
         if (this.state.analysis !== null) {
             const {  // object destructuring:
-                total_view_time,
+                total_view_time, all_responses
```

Comment:
just to keep things sane, bc we're gonna end up having a ton of these, one per line with a trailing comma plz.